tests/mysql_insertion/data.csv

```diff
@@ -0,0 +1,15 @@
+1,Bella,Cat,3000
```

Comment:
I think we need to add this and Config.toml to  Cloud.toml 
Sample - https://github.com/ballerina-platform/module-ballerina-c2c/blob/master/samples/sample5

Reply:
Updated